capture-sdk/sdk/src/doc/source/migrate-to-3-0-0.rst

```diff
@@ -318,3 +318,29 @@ There is a default implementation for indicating that document analysis is in pr
 by implementing the ``CustomLoadingIndicatorAdapter`` interface and passing it to ``GiniCapture``.
 
 You can find more details `here <features.html#analysis-screen-customization>`_.
+
+Error
+~~~~~
+
+The new error screen uses the global UI customization options.
+
+Breaking Changes
+++++++++++++++++
+
+Showing errors during usage of the SDK was changed from snackbar to a whole new screen.
+
+New Features
+++++++++++++
+
+New UI
+^^^^^^
+
+The new error screen gives options to retake photos or enter details manually and displays errors with more detailed description.
```

Comment:
[Request] Same as in PR #186 we should mention the enter manually result code as a separate new feature. You can mention it here and just add a link to the features page.